source/optional_features/custom_metadata/index.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Custom metadata
+weight: 135
+---
+
+# Custom metadata
+
+You can add custom metadata to a new payment. For example, you can add a reference number from your finance or accounting system, so you can reconcile the payment against bank transaction information.
```

Comment:
The reference to bank transaction information is potentially misleading. The metadata won't end up on the bank statement. The feature is more about giving the ability to 'post' payments to fund/ledger codes within income management systems and general ledger systems.

Reply:
Ah thanks for the clarification. Would just `... reconcile the payment later.` be accurate?

Reply:
yes it would be